server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -2002,42 +1559,28 @@ public AsyncJobStatusDTO undoImports(
         JobConfig config = UndoImportsJob.createJobConfig()
             .setOwner(owner);
 
-        AsyncJobStatus job = this.jobManager.queueJob(config);
+        AsyncJobStatus job;
```

Comment:
Same questions here

Reply:
The auto-generated method does not throws an exception.